gnome-extra/chrome-gnome-shell/chrome-gnome-shell-7.ebuild

```diff
@@ -19,15 +19,26 @@ IUSE=""
 
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
+DEPEND="${PYTHON_DEPS}
+	sys-apps/coreutils
+"
 RDEPEND="${PYTHON_DEPS}
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
 	gnome-base/gnome-shell
 "
 
 src_configure() {
 	python_setup
```

Comment:
dont't need this, `python-single-r1` exports its `pkg_setup` which does the setup.